### DISCUSSION TITLE
Configure MacOS Platform SSO

### DISCUSSION THREAD

**Discussion by giscus** [ugurkocde/intunemacadmins#53] [General]
# Configure MacOS Platform SSO

Comprehensive guide on configuring MacOS Platform SSO (Single Sign-On) with Secure Enclave Key. Learn about prerequisites, policy creation, and important considerations for implementation.

https://www.intunemacadmins.com/complete-guide-macos-deployment/configure_macos_platform_sso/

**Comment by TorAndreSolbakken** [comment]
Really nice post, have anyone tried to use both password and UserSecureEnclaveKey in 2 different policies to enable both? It would be nice to have the windows experience an flexibility.

**Comment by TheOtherNeo** [comment]
Do I need to apply this policy to "all users" or "all devices"? The OIB is listed as "D" implying that Platform SSO policy need to be applied to "all devices", however I found that registraiton fails. In the Intune documentation, it is somewhat implied that it needs to be applied to users and will fail if applied to the device.

**Reply by TorAndreSolbakken** [reply]
I did deploy it to a device collection that contains devices with a dynamic query based on MDM enrollment.

And it works as expected.

**Comment by TheOtherNeo** [comment]
This is what I found between all the documents: "For Platform SSO settings on devices with user affinity, it's not supported to assign to device groups or filters. When using device group assignment or user group assignment with filters on devices with user affinity, the user might be unable to access resources protected by Conditional Access."

**Comment by chriswhitingBCH** [comment]
Anyone else come across this where the setup assistant doesnt prompt to create the new local user? Think the issue is applying the Platform SSO at the user level, but that is what Microsoft recommend ? https://github.com/ugurkocde/intunemacadmins/issues/69
